.README/rules/sort-tags.md

```diff
@@ -8,24 +8,32 @@ Sorts tags by a specified sequence according to tag name.
 
 ##### `tagSequence`
 
-An array of tag names indicating the preferred sequence for sorting tags.
+An array of tag group objects indicating the preferred sequence for sorting tags.
+
+Each item in the array can be an object with a `tags` property set to an array
+of tag names.
 
 Tag names earlier in the list will be arranged first. The relative position of
 tags of the same name will not be changed.
 
+Earlier groups will also be arranged before later groups, but with the added
+feature that additional line breaks may be added between (or before or after)
+such groups.
+
 Tags not in the list will be sorted alphabetically at the end (or in place of
```

Comment:
I think this paragraph could be rephrased. Is this an improvement to get better flow?

"Tag names not in the list will be grouped together at the end. The psuedo-tag `-other` can be used to place them anywhere else if wanted. The tags will be placed in their order of appearance, or alphabetized if the `alphabetizeExtras` is enabled, see more below about that option."

Reply:
Yes, thanks, I have borrowed from this in a new commit.